libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/LocationUpdater.kt

```diff
@@ -0,0 +1,82 @@
+package com.mapbox.services.android.navigation.v5.navigation
```

Comment:
when are we going to `ktlint` new code?

Reply:
cc @RingerJK

Reply:
already in [PR](https://github.com/mapbox/mapbox-navigation-android/pull/2070)